test/core/fixtures/transformation/es7.function-bind/call/expected.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+var _context;
+
+ns.obj.func.call(ctx);
+(_context = ns.obj).func.call(_context);
+
+ns.obj1.func.call(ns.obj2);
```

Comment:
I don't really think this matters in 99% of cases unless someone specifically sets getter, for example, in order to log accesses to functions, do you?